src/app/records/presentation-nodes.ts

```diff
@@ -79,13 +95,20 @@ export interface DimPresentationNodeSearchResult extends DimPresentationNodeLeaf
 /** Process the live data into DIM types that collect everything in one place and can be filtered/searched. */
 export function toPresentationNodeTree(
   itemCreationContext: ItemCreationContext,
-  node: number
+  node: number,
+  plugSetCollections?: { hash: number; displayItem: number }[]
 ): DimPresentationNode | null {
   const { defs, buckets, profileResponse } = itemCreationContext;
   const presentationNodeDef = defs.PresentationNode.get(node);
   if (presentationNodeDef.redacted) {
     return null;
   }
+  const commonNodeProperties = {
+    presentationNodeDef,
```

Comment:
Doesn't this need to be:
```suggestion
    nodeDef: presentationNodeDef,
```

Reply:
Good catch, types adjusted and fixed.